### ISSUE TITLE
[AdminBundle]: "Add rule" in shipping methods no longer opens full rule form

### ISSUE THREAD

**Issue by sabat24** [Sylius/Sylius#14009]
**Sylius version affected**: 1.1.11

**Description**  
Same bug as https://github.com/Sylius/Sylius/issues/13463 but for Shipping Methods.

**Possible Solution**  

Add below code to `Resources/private/js/app.js` somewhere inside `$(document).ready(() => {`

```
$('#sylius_shipping_method_rules > a[data-form-collection="add"]').on('click', () => {
    setTimeout(() => {
      $('select[name^="sylius_shipping_method[rules]"][name$="[type]"]').last().change();
    }, 50);
  });
```

**Side Note**

There should be more generic solution to handle such things not directly related to field names nor ids. It would help us when we extending some forms and would avoid such bugs from your side each time you change field name or add something.

**Comment by Ferror** [comment]
Hi @sabat24 ! thanks for the issue.

I looked at it and could not reproduce the issue under https://master.demo.sylius.com/admin/shipping-methods/ (maybe it's already fixed or sth) Do you mind sending us ex. loom video (https://www.loom.com/) 😄

**Comment by sabat24** [comment]
@Ferror 

Go to https://master.demo.sylius.com/admin/shipping-methods/new and click on `Add rule` button. Then you will see

![obraz](https://user-images.githubusercontent.com/6381661/170224380-c0978489-441b-4843-932b-d76ac7c56777.png)

instead of

![obraz](https://user-images.githubusercontent.com/6381661/170224490-ae183e9c-1f1c-4dcd-bf42-0321a16a9171.png)

As you can see in first screenshot there is no configuration fields for the rule.

**Comment by Ferror** [comment]
Alright, now I see it. The first time it is chosen the input does not appear. Would you like to open a PR with the improvement?

**Comment by sabat24** [comment]
Sure. Will do it in few days as soon as I write Behat tests for it.

**Comment by jakubtobiasz** [comment]
Hello 👋🏼,
this issue has been resolved by #14730.
